target/linux/ipq40xx/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -9,8 +9,11 @@ board=$(board_name)
 case "$FIRMWARE" in
 "ath10k/cal-pci-0000:01:00.0.bin")
 	case "$board" in
+	meraki,mr30h |\
 	meraki,mr33 |\
-	meraki,mr74)
+	meraki,mr74 |\
+	meraki,z3 |\
```

Comment:
Z3 and Z3C don't have the third radio, so this can be skipped. Not sure about others, as I don't have them (yet).